### PR TITLE
gcloud-*: add pages

### PR DESCRIPTION
**Pull request by FarhadK2022**
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by sbrl** [comment]
Once all of the linter errors are fixed, we can begin with the review process, @FarhadK2022 :-)

**Comment by FarhadK2022** [comment]
still working on this. something is going on with branch tracking

**Comment by kbdharun** [comment]
I am making changes to this branch locally, please don't push any new changes for a few minutes.

File: pages/common/gcloud-app.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# gcloud app
+
+> Build highly scalable applications on a fully managed serverless platform.
```
Comment (sbrl) [inline pages/common/gcloud-app.md:3]: This seems subjective.
```suggestion
> Build scalable applications on a managed serverless platform.
```

File: pages/common/gcloud-app.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> See also: `gcloud`.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/app>.
+
+- Deploy your app's code and configuration to the App Engine server:
```
Comment (sbrl) [inline pages/common/gcloud-app.md:7]: Avoid you/your.
```suggestion
- Deploy an app's code and configuration to the App Engine server:
```

File: pages/common/gcloud-app.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- List all versions of all services deployed to the App Engine server:
+
+`gcloud app versions list`
+
+- Open the current app in a web browser:
+
+`gcloud app browse`
+
+- Create an App Engine app within your current project:
```
Comment (sbrl) [inline pages/common/gcloud-app.md:19]: Avoid you/your.
```suggestion
- Create an App Engine app within the current project:
```

File: pages/common/gcloud-auth.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Authorize Google Cloud access for the `gcloud` CLI with Google Cloud user credentials and set the current account as active:
+
+`gcloud auth login`
+
+- Authorize Google Cloud access similar to `gcloud auth login` but with service account credentials:
+
+`gcloud auth activate-service-account`
+
+- Manage your Application Default Credentials (ADC) for Cloud Client Libraries:
```
Comment (sbrl) [inline pages/common/gcloud-auth.md:15]: Again with the you/your
```suggestion
- Manage Application Default Credentials (ADC) for Cloud Client Libraries:
```

File: pages/common/gcloud-auth.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`gcloud auth application-default`
+
+- List all credentialed accounts:
```
Comment (sbrl) [inline pages/common/gcloud-auth.md:19]: Can accounts be 'uncredentialed'? Might need rewording here?

File: pages/common/gcloud-config.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# gcloud config
+
+> Personalize your `gcloud` configuration with properties.
```
Comment (sbrl) [inline pages/common/gcloud-config.md:3]: Avoid you/your.
```suggestion
> Manage different configurations of `gcloud`. 
```

File: pages/common/gcloud-config.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- Fetch the value of a `gcloud` property:
+
+`gcloud config get {{property}}`
+
+- Display all the properties for the current configuration:
+
+`gcloud config list`
+
+- Create a new named configuration:
```
Comment (sbrl) [inline pages/common/gcloud-config.md:19]: Perhaps this will read better?
```suggestion
- Create a new configuration with a given name:
```

File: pages/common/gcloud-config.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+`gcloud config configurations create {{configuration_name}}`
+
+- Display a list of all available configurations:
+
+`gcloud config configurations list`
+
+- Switch to an existing named configuration:
```
Comment (sbrl) [inline pages/common/gcloud-config.md:27]: Perhaps this will read better?
```suggestion
- Switch to an existing configuration with a given name:
```

File: pages/common/gcloud-feedback.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+
+> Provide feedback to the Google Cloud team.
+> See also: `gcloud`.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/feedback>.
+
+- Provide feedback to the `gcloud` team:
+
+`gcloud feedback`
+
+- Provide feedback to the `gcloud` team, including a log file:
```
Comment (sbrl) [inline pages/common/gcloud-feedback.md:11]: Perhaps this will read better?
```suggestion
- Provide feedback to the `gcloud` team and attach a log file:
```

File: pages/common/gcloud-help.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+# gcloud help
+
+> Display help and reference information for `gcloud`.
+> For supplementary help for topics not directly associated with individual commands, please take a look at `tldr gcloud topic`.
```
Comment (sbrl) [inline pages/common/gcloud-help.md:4]: Perhaps this will read better?
```suggestion
> For supplementary help for topics not directly associated with individual commands, see also `tldr gcloud topic`.
```

File: pages/common/gcloud-topic.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
+# gcloud topic
+
+> Provides supplementary help for topics not directly associated with individual commands. See also `gcloud`.
+> For general help, please take a look at `tldr gcloud help`.
```
Comment (sbrl) [inline pages/common/gcloud-topic.md:4]: Perhaps this will read better?
```suggestion
> For general help, see `tldr gcloud help`.
```

File: pages/common/gcloud.md [hunk 12]
```diff
@@ -1,13 +1,14 @@
 # gcloud
 
 > The official CLI tool for Google Cloud Platform.
+> Some subcommands such as `gcloud iam` have their own usage documentation.
```
Comment (sbrl) [inline pages/common/gcloud.md:4]: The eventual aim is to document all subcommands. Can we reword this to be more generic?

File: pages/common/gcloud.md [hunk 13]
```diff
@@ -1,13 +1,14 @@
...
 > More information: <https://cloud.google.com/sdk/gcloud>.
 
 - List all properties in one's active configuration:
 
 `gcloud config list`
 
-- Log in to Google account:
+- Login to Google account:
```
Comment (sbrl) [inline pages/common/gcloud.md:11]: Perhaps this will read better?
```suggestion
- Login to a Google account:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new pages! I've left some comments below for you to review.

@kbdharun: Try to avoid making changes directly to other peoples' PRs unless it's clear the PR has been abandoned (say, after 1-2 weeks). That way authors can learn :-)

**Review by sbrl — CHANGES_REQUESTED** [review]
Marking as request changes ref my review

**Comment by FarhadK2022** [comment]
Just checking in, as I am new to this. Is there anything else that needs to be done on my end?

**Comment by kbdharun** [comment]
> Just checking in, as I am new to this. Is there anything else that needs to be done on my end?

Feel free to apply and address @sbrl's suggestions above, sorry for initially making changes to PR without permission (when you were active) had some drafts locally for some `gcloud` commands.

File: pages/common/gcloud.md (hunk 12 above)
Comment (kbdharun) [reply pages/common/gcloud.md:4]: Sure

```suggestion
> Note: `gcloud` subcommands have their own usage documentation.
```

File: pages/common/gcloud-auth.md (hunk 5 above)
Comment (kbdharun) [reply pages/common/gcloud-auth.md:19]: We used the documentation's description here, will update it to something like this:

```suggestion
- Display a list of Google Cloud accounts currently authenticated on your system:
```

File: pages/common/gcloud-config.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
+# gcloud config
+
+> Manage different configurations of `gcloud`.
```
Comment (kbdharun) [inline pages/common/gcloud-config.md:3]:
```suggestion
> Manage different configurations of `gcloud`.
```

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
> GTG?

Yeah, we can perform any additional fixes/changes separately if necessary.
